app/Resources/views/themes/tastefulplanet/fields/ezrichtext/custom_tags/ezfacebook.html.twig

```diff
@@ -0,0 +1,14 @@
+<script>(function(d, s, id) {
+        var js, fjs = d.getElementsByTagName(s)[0];
+        if (d.getElementById(id)) return;
+        js = d.createElement(s); js.id = id;
+        js.src = 'https://connect.facebook.net/en_EN/sdk.js#xfbml=1&version=v2.11';
+        fjs.parentNode.insertBefore(js, fjs);
+    }(document, 'script', 'facebook-jssdk'));
+</script>
+
+<div
+        class="fb-post" style="display: {{ params.display }};"
+        data-href="{{ params.post_url }}"
+        data-width="{{ params.width ? params.width : '' }}"
```

Comment:
Did you check the issue I found in ezplatform-ee-demo with empty width?

Reply:
Yes, addressed in [5b17535](https://github.com/ezsystems/ezplatform-demo/commit/5b17535465ddc9cf75d4baec624b7458caf31c7f#diff-518cad18e7ba0a166286fa1fa2a5b683R5), my mistake.